### ISSUE TITLE
Download example isn't working

### ISSUE THREAD

**Issue by astroice7** [seignovert/pyvims#3]
Hello, 
I am having trouble trying to use _wget_ to download a VIMS .cub file with Windows 10 OS, but it appears that the data in the ReadMe example is inaccessible (see bullet 1). I tried to use the following commands in the Windows Powershell and Python 3.6 respectively:

- `> wget https://vims.univ-nantes.fr/data/isis/T3/C1487096932_1_vis.cub` 
_wget : Not Found
The requested URL /data/isis/T3/C1487096932_1_vis.cub was not found on this server._
I receive the following error and checked the url to see that I wasn't allowed access to these folders, which makes this downloading method unusable.

- `import wget; wget.download("https://vims.univ-nantes.fr/data/isis/T3/C1487096932_1_vis.cub")` 
_urllib.error.HTTPError: HTTP Error 404: Not Found_
In Python, this raises a similar error for the same example file and confirms the denied access to those folders. 

- `wget.download("https://www.vims.univ-nantes.fr/ir/cal/N1790066425_1_ir.cub")`
_ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it_
I saw another link to download the individual C*_ir.cub and N*_ir.cub files in the [ISIS3 data pipeline webpage](https://vims.univ-nantes.fr/info/isis-calibration) and receive a more workable error. I tried to resolve it by editing my proxy server settings to automatic as [described in StackOverflow](https://stackoverflow.com/questions/20437701/winerror-10061-no-connection-could-be-made), but it didn't change anything. I will continue to explore a solution to this error for the meantime as it might be something on my end. 

Nonetheless, is there any way that you could fix the download example to allow access to the .cub files in the _\flyby_ folders? Downloading them individually from the cube pages would be cumbersome. Thanks!

**Comment by seignovert** [comment]
Hi @astroice7,

Thank you for your feedback. Your configuration seems to be fine. The problem was on our side.
We recently updated the way we manage the data on the [vims.univ-nantes.fr](https://vims.univ-nantes.fr) website.

In the initial version, the data were stored with the `targeted` flyby key (`T3` in this case). However we now provide data for `targeted` and `non-target` flybys. Since `non-targeted` cubes don't have a `targeted` key, we switched to `non-targeted` keys for all the cubes. In this case, you only need to change `T3` by `003TI` in the url and you will get the expected files.

You could also download manually the files on the [cube page](https://vims.univ-nantes.fr/cube/1487096932_1) by using the cyan buttons <kbd>VIS.cub</kbd> and <kbd>IR.cub</kbd> at the bottom (2 files will be downloaded per click).
Or, as you mentioned in the [ISIS3 data pipeline description](https://vims.univ-nantes.fr/info/isis-calibration), if you want to use `wget` but you don't know the `non-targeted` key you can use these shortcuts:
```bash
wget https://vims.univ-nantes.fr/vis/cal/1487096932_1 -O C1487096932_1_vis.cub
wget https://vims.univ-nantes.fr/ir/cal/1487096932_1  -O C1487096932_1_ir.cub
wget https://vims.univ-nantes.fr/vis/nav/1487096932_1 -O N1487096932_1_vis.cub
wget https://vims.univ-nantes.fr/ir/nav/1487096932_1  -O N1487096932_1_ir.cub
```
 The only drawback is that you need to specify the output filename (with `-O` option) otherwise all your files will be labeled `1487096932_1`.

This package is still under a `beta` status, so feel free to open a new issue if needed.

**Comment by seignovert** [comment]
Fix #3 with 757dc14ca067d939247f956accf13e87290da4da
